packages/wrangler/src/deploy/index.ts

```diff
@@ -269,11 +283,43 @@ export const deployCommand = createCommand({
 			);
 		}
 
+		const argsContainsBothPathAndScript = args.path && args.script;
+
+		if (argsContainsBothPathAndScript) {
+			// Note: The following logic is as is because changing it to something different would be a breaking change
+			//
+			// TODO: In the next Wrangler major change this logic to treat the path arg as an assets directory while
+			//       using the script arg as the entrypoint path (e.g. just as `wrangler deploy --assets <ASSETS_PATH> --script <SCRIPT_PATH>`)
```

Comment:
Would this really be a breaking change?
The use must have opted in to providing the `--script` arg, so they can expect different behaviour, no?